package.json

```diff
@@ -34,11 +34,11 @@
   },
   "homepage": "https://github.com/nhsuk/profiles#readme",
   "dependencies": {
-    "body-parser": "^1.17.2",
-    "compression": "^1.7.0",
+    "body-parser": "1.18.2",
```

Comment:
Is there any reason the caret has been removed from the version numbers for the updated packages?

Reply:
not intentionally, I assume that's `yarn upgrade`'s handywork